dialogue-target/src/main/java/com/palantir/dialogue/Clients.java

```diff
@@ -24,12 +24,21 @@
  * duplicating logic.
  */
 public interface Clients {
+    /**
+     * Makes a request to the specified {@link EndpointChannel} and deserializes the response using a provided
+     * deserializer.
+     */
+    <T> ListenableFuture<T> call(EndpointChannel channel, Request request, Deserializer<T> deserializer);
 
     /**
      * Makes a request to the specified {@link Endpoint} and deserializes the response using a provided deserializer.
+     * @deprecated prefer {@link #bind} as this allows pre-computing values to save CPU
      */
+    @Deprecated
```

Comment:
This may cause issues compiling generated code before the generator is upgraded. May want to hold off a day or two before adding a deprecation.